docs/user/setup.rst

```diff
@@ -56,6 +56,7 @@ Native has been used with:
 
     $ brew install llvm
     $ brew install bdw-gc re2 # optional
+    $ brew install clang-format # required if using ./bin/clangfmt
```

Comment:
User guide mentions things necessary to use the latest released version of Scala Native, not build and develop it from source. Putting clang format in there might be confusing as it's not necessary from the end user point of view.